From 2852e4605c4c6d215183203834fcf4ebfed37efd Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Fri, 31 Jul 2020 18:04:16 +0700
Subject: [PATCH 424/504] ARM: dts: r8a7742: Add vsp nodes to SoC dtsi

This patch adds description for VSP modules (Video Signal Processing) of
R8A7742 SoC (RZ/G1H).

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index 1d26f53..dc2f761 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -118,6 +118,22 @@
 		};
 	};
 
+	vspm@fe920000 {
+		compatible = "renesas,vspm-vspr", "renesas-vspm";
+		reg = <0 0xfe920000 0 0x8000>;
+		interrupts = <0 266 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7742_CLK_VSP1_S>;
+		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
+	};
+
+	vspm@fe928000 {
+		compatible = "renesas,vspm-vsps", "renesas-vspm";
+		reg = <0 0xfe928000 0 0x7404>;
+		interrupts = <0 267 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&mstp1_clks R8A7742_CLK_VSP1_S>;
+		power-domains = <&sysc R8A7742_PD_ALWAYS_ON>;
+	};
+
 	vspd0: vspd0@fe930000 {
 		compatible = "renesas,vsp2";
 		reg = <0 0xfe930000 0 0x7404>;
-- 
2.7.4

